dist_run.py

```diff
@@ -41,10 +40,17 @@ def _create_device_mesh(mesh_dimensions):
     return dist.init_device_mesh("cuda", mesh_dimensions, mesh_dim_names=("pp", "tp"))
 
 
-def _load_model_weights(stage_module, hf_model_name, device, logger):
+def _load_model_weights(stage_module, hf_model_name, device, logger, model_config):
```

Comment:
nit: remove logger from function signature?
nit: comment on what `model_config` is purposed for?

Reply:
resolved both!